ddtrace/llmobs/_llmobs.py

```diff
@@ -97,14 +102,32 @@
 }
 
 
+@dataclass
+class LLMObsSpan:
```

Comment:
not sure if this would benefit from some typedocs here - i do feel it's pretty self-explanatory so might not be necessary, but up to you!

Reply:
yeah i was midway through documenting and questioned what i would meaningfully write that isn't described by the fields. Maybe we could include an example usage? Might be worth saying that the fields are mutable 🤷‍♂️

Reply:
yeah i think an example usage would be good enough for this, yeah the fields themselves are self-explanatory 😂